src/gui/src/heatMap.cpp

```diff
@@ -105,8 +105,6 @@ void HeatMapDataSource::dumpToFile(const std::string& file)
 
 void HeatMapDataSource::redraw()
 {
-  ensureMap();
```

Comment:
Why is this no longer required?

Reply:
when the render starts it calls `ensureMap()` so this only serves to rebuild the map immediately instead of when it's needed. Because this unsets the destroy_map flag, we end up repeatedly destroying and rebuilding the map